.genx/prompts/ui.js

```diff
@@ -10,13 +10,23 @@ module.exports = async (inquirer, prevAns = {}) => {
     {
       name: 'routes',
       type: 'input',
-      message: 'Pages in comma separated format e.g. home,dashboard,admin',
+      message: 'Pages config in json format',
       when: !prevAns.routes,
-      default: prevAns.routes || 'home',
+      default: `[{name: 'home'}]`,
     },
   ])
 
+  let routesParsed = [{name: 'home'}];
+  if (routes) {
+    try {
+      routesParsed = JSON.parse(routes);
+      console.log(routesParsed)
```

Comment:
We can remove this log - we'll know it was valid if there's no error

Reply:
yeah, looks like leftover from debugging here